force-app/main/default/classes/CalendarWorkOrdersController.cls

```diff
@@ -1,9 +1,16 @@
 public without sharing class CalendarWorkOrdersController {
     @AuraEnabled(cacheable=true)
-    public static List<SObject> getObjects(String param) {
-        String variable = param;
+    public static List<SObject> getObjects(String record, String start, String endTime) {
+        String variable = record;
+        String startDate = start;
+        String endDate = endTime;
         List<SObject> objects = (List<SObject>) Database.query(
-            'SELECT Id, StartDate, EndDate, Subject FROM ' + variable
+            'SELECT Id, ' +
+            startDate +
+            ', ' +
+            endDate +
+            ', Subject FROM ' +
```

Comment:
Subject heter ikke alltid subject. Må ta inn som argument, akkurat som for startDate/endDate

Reply:
Løst